server/graphql/preference/src/lib.rs

```diff
@@ -48,6 +48,7 @@ impl PreferenceQueries {
         ctx: &Context<'_>,
         store_id: String,
         pref_type: PreferenceNodeType,
+        context: PreferenceDescriptionContext,
```

Comment:
👀 Had to double take here... We're using context in a few places with various meanings... Could possible make this clearer? 👀

Reply:
Ah yeah sure! will rename..